tests/testthat/test-send_sparql.R

```diff
@@ -1,4 +1,5 @@
 test_that("send_sparql() returns tibble", {
+  skip_if_offline()
```

Comment:
WikidataQueryServiceR uses httr so I'd have to use httptest, not httptest2 for mocking.